sphinxlint/checkers.py

```diff
@@ -485,6 +485,38 @@ def check_block(block_lineno, block):
     yield from errors
 
 
+_find_leading_spaces = re.compile(r'^\s*').match
+# finds lines that start with bullets, numbers, and directives' ".."
+_find_list_starters = re.compile(r'^\s*(?:[-+*•‣⁃]|\d+[).]|\(\d+\)|#\.|\.\.)\s+'
+                                 r'(?!index)').match
+
+
+@checker(".rst")
+def check_excessive_indentation(file, lines, options=None):
```

Comment:
The previous checker is called `bad_dedent`, so this could be called `bad_indent`.  However `indent` sounds more generic since it might refer to both indentation and dedentation, whereas this checker only deals with excessive indentation.